docs/UserGuide.md

```diff
@@ -339,6 +341,13 @@ Exports a CSV file of the current view containing specified fields.
 
 Notes:
 * Name is always the first column in the CSV file.
+* Including any of the arguments will export include that field in the CSV file. 
```

Comment:
I think `Including any of the arguments will include that field in the exported CSV file.` is better

Reply:
Agree